### PR TITLE
Prepare release 0.13.1.0

### PR DESCRIPTION
**Pull request by Shimuuar**
As promised in #467. Bump version and changelog update

### CODE REVIEW

**Comment by Bodigrim** [comment]
LGTM. I suggest we wait couple more days for `primitive-0.9` (https://github.com/haskell/primitive/issues/378) to ensure compatibility with it.

File: vector/changelog.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
+# Changes in version 0.13.1.0
+
+ * Specialized variants of `elemIndexR` are reexported for all vector
```
Comment (lehins) [inline vector/changelog.md:3]:
```suggestion
 * Specialized variants of `findIndexR` are reexported for all vector
```

**Review by lehins — APPROVED** [review]
:+1:

**Comment by Vekhir** [comment]
primitive 0.9.0.0 has been released

**Review by Bodigrim — APPROVED** [review]
Let's go!

**Comment by Vekhir** [comment]
Just tested, and 0.13.0.0 builds and tests fine with primitive 0.9.0.0. Are you going to include the version bump?

**Comment by Shimuuar** [comment]
Done. I also fixed deprecation warnings in CI

**Comment by Bodigrim** [comment]
@Shimuuar I granted you Hackage rights, feel free to upload.

**Comment by Shimuuar** [comment]
OK. I'll made upload tomorrow

**Comment by Shimuuar** [comment]
Done 0.13.1.0 is on hackage

**Pull request merged**